.circleci/config.yml

```diff
@@ -648,6 +659,23 @@ jobs:
               docker push taraxa/${IMAGE}:${START_TIME}-${CIRCLE_SHA1}
               docker push taraxa/${IMAGE}:latest
            fi
+
+      - run:
+          name: Verify Helm chart with linter
+          command: |
+            helm lint charts/${HELMCHART}/
+
+      - run:
+          name: Generate Helm chart and output yaml manifests
+          command: |
+            helm dependency build charts/${HELMCHART}/
+            helm template charts/${HELMCHART}/ --values charts/${HELMCHART}/values.yaml
```

Comment:
Why do we need this?

If it's to verify if template is working, thanks it's fine! but we should add to the `run` name

Reply:
Yes. This is just to verify.

What do you mean to add `run`?

Reply:
`name: Generate Helm chart and output yaml manifests for verification`

Or something like this, so people reading knows it's only to verify if it is working, it wasn't clear for me at first glance
I am really bad with names haha

Reply:
updated this task name